waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/server/FederatedHMSHandlerTest.java

```diff
@@ -767,12 +768,42 @@ public void flushCache() throws TException {
 
   @Test
   public void get_all_functions() throws MetaException, TException {
```

Comment:
remove this test it is not a good test anymore.

Reply:
done